pkg/perf/healthcheck/healthcheck.go

```diff
@@ -62,7 +62,7 @@ func (h *healthcheckTask) Run(ctx context.Context) (interface{}, error) {
 	log.Debug().Msg("starting health check task")
 	for k := range h.errors {
 		// reset errors on each run
-		h.errors[k] = nil
```

Comment:
This will cause the errors on ZUI to be permanent since we loop over errors map and send the label with its errors. If the label was removed it won't be set in case of no errors so it won't be sent to ZUI for resetting.